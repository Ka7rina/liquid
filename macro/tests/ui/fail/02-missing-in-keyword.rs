use liquid_macro::seq;

seq!(in 1..10 {
    // nothing
});

fn main() {}
