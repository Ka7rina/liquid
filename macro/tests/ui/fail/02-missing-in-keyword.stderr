error: expected `in`
 --> $DIR/02-missing-in-keyword.rs:3:9
  |
3 | seq!(in 1..10 {
  |         ^
